uk.co.compucorp.civicrm.civihrcore/CRM/CiviHRCore.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class CRM_CiviHRCore {
```

Comment:
Okay so I assume that we could leave CRM/CiviHRCore namespace as it is, but move the static method into Info class (or something) - CRM_CiviHRCore_Info. Would it be fine?